src/components/ZoneCard.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div class="zone">
+    <div class="container">
+      <div class="temp">
+        <h6>Temperature</h6>
+        <h1 id="currentTemp">{{ state.Temperature }}Cº</h1>
+        <h4>Min: {{ state.minTemp }}Cº</h4>
+        <h4>Max: {{ state.maxTemp }}Cº</h4>
+      </div>
+      <div class="humid">
+        <h6>Humidity</h6>
+        <h1 id="currentHumid">{{ state.Humidity }}%</h1>
+        <h4>Min: {{ state.minHumid }}%</h4>
+        <h4>Max: {{ state.maxHumid }}%</h4>
+      </div>
+    </div>
+    <h2>{{ title }}</h2>
+    <slot />
+  </div>
+</template>
+
+<script setup>
+import { defineProps, reactive } from "vue";
+
+defineProps({
+  title: String
+});
+
+const state = reactive({
+  Temperature: 40,
+  Humidity: 32,
+  maxTemp: 41,
+  minTemp: 12,
+  maxHumid: 43,
+  minHumid: 28
```

Comment:
The casing in the object properties (keys) is a little inconsistent here. The general convention is to use `lowerCamelCase`

Reply:
Yes, you are right. I was aware of this and should have fixed this already.